cime_config/tests.py

```diff
@@ -256,7 +263,7 @@
         },
 
     "e3sm_developer" : {
-        "inherit" : ("e3sm_land_developer", "e3sm_atm_developer", "e3sm_ice_developer"),
+        "inherit" : ("e3sm_land_developer", "e3sm_atm_developer", "e3sm_ice_developer", "e3sm_landice_developer"),
         "time"    : "0:45:00",
```

Comment:
Take this out of e3sm_developer for now since it has to run with gnu.

Reply:
Done. Thanks.